packages/salesforcedx-vscode-core/test/vscode-integration/orgPicker/orgList.test.ts

```diff
@@ -66,14 +66,14 @@ describe('orgList Tests', () => {
       readFileStub.callThrough();
       const orgList = new OrgList();
 
-      const authInfoObjects = await orgList.getAuthInfoObjects();
-      if (authInfoObjects) {
-        expect(authInfoObjects.length).to.equal(2);
+      const orgAuthorizations = await orgList.getOrgAuthorizations();
+      if (orgAuthorizations) {
```

Comment:
the if in this test weirds me out

Reply:
Hmm.  Yeah, might need to re-work these tests since the orgList is now generated using the call to AuthInfo.listAllAuthorizations() and the OrgAuthorization type instead of FileInfo.

Reply:
I created a work item to track: [W-11528882](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001313qkYAA/view)